test/example-types/results.js

```diff
@@ -164,7 +164,7 @@ describe('results', () => {
     ])
   })
   it('should sort on "_score: desc" with no sortField config', async () =>
```

Comment:
Looks like we can drop the `async` from this test since it seems to be synchronous now.